README.md

```diff
@@ -58,6 +58,8 @@ The following are the different environment variables that are looked up that al
 | ASYNC_SCHEMA_UPDATES  | Specifies if async achema updates is enabled                                  | false                     |
 | KAFKA_BROKER_HOST     | Host name of the Kafka broker, used if ASYNC_SCHEMA_UPDATES = true            | gql-schema-registry-kafka |
 | KAFKA_BROKER_PORT     | Port used when connecting to Kafka, used if ASYNC_SCHEMA_UPDATES = true       | 9092                      |
+| LOG_LEVEL             | Minimum level of logs to output                                               | info                      |
+| LOG_TYPE              | Output log type, provided by tslog. Supports pretty, json, hidden.            | pretty                    |
```

Comment:
not sure this is needed, I would ideally prefer something like [winston](https://www.npmjs.com/package/winston ) instead of tslog to not only format but also transport logs somewhere